### ISSUE TITLE
Failed to instantiate module angular-meteor

### ISSUE THREAD

**Issue by sstativa** [idanwe/ionic-cli-meteor-whatsapp-tutorial#3]
Hi there,

I was trying to follow the tutorial and build app using `ionic cli`, but it does not work for me. I'm getting following errors:

```
angular-meteor.bundle.js:1142
Uncaught TypeError: Cannot read property 'ECMAScript' of undefined

ionic.bundle.js:13212 
Uncaught Error: [$injector:modulerr] Failed to instantiate module whatsapp due to:
Error: [$injector:modulerr] Failed to instantiate module angular-meteor due to:
Error: [$injector:nomod] Module 'angular-meteor' is not available! You either misspelled the module name or forgot to load it. If registering a module ensure that you specify the dependencies as the second argument.
```

I guess the 2nd error is result of 1st one, but I cannot figure out how to include `ecmascript` into the project.

**Comment by idanwe** [comment]
It is a new dependency of angular-meteor that need to be bundled. 
I will create PR for that

**Comment by daengdeni** [comment]
I have same problem.. how to resolve this?

**Comment by idanwe** [comment]
@sstativa angular-meteor released 1.2 with [the fix](https://github.com/Urigo/angular-meteor/pull/844)
